in_toto/models/layout.py

```diff
@@ -135,13 +137,26 @@ def import_step_metadata_from_files_as_dict(self):
       Calls functions to read files from disk
 
     <Returns>
-      A dictionary with Link names as keys and Link objects as values.
+      A dictionary with Link names as keys and a dict (key_id Link objects)
+      as values.
 
     """
     step_link_dict = {}
     for step in self.steps:
-      link = models__link.Link.read_from_file(step.name + '.link')
-      step_link_dict[step.name] = link
+      key_link_dict = {}
+      link_count = 0
+      for keyid in step.pubkeys:
+        link_count += 1
+        try:
+          link = models__link.Link.read_from_file(FILENAME_FORMAT.format(
+            step_name=step.name, keyid=keyid))
```

Comment:
We use [double-indentation for line continuation](https://github.com/secure-systems-lab/code-style-guidelines#indentation).